docs/API.md

```diff
@@ -562,16 +562,20 @@ All conversation's participants who is online will receive the following message
     id: "63480e68f4794709f802a2fa",
     t: 15673838833,
     from: "634ec51c0b65918393dca5bf",
+    title: "UserName",
```

Comment:
`title` and `conversation_type` shoudl not be a part of message (only push notification).
A user already have a conversation object locally from where he/she can extract this info.